File: PHP/uberX.php
<?php
require_once('car.php');
class UberX extends Car {
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model) {            
        parent::__construct($license, $driver); // acceder a la clase padre con la palabra reservada parent
        $this->brand = $brand;
        $this->model = $model;
    }
}
?>

File: Java/UberX.java
class UberX extends Car { // herencia es con extens
    String brand;
    String model;

    public UberX(String license, Account driver, String brand, String model) {
        super(license, driver); // super es para referirnos al método constructor de la clase padre
        this.brand = brand;
        this.model = model;
    }

}